scaleway/resource_rdb_instance_test.go

```diff
@@ -263,34 +283,6 @@ func testAccCheckScalewayRdbExists(tt *TestTools, n string) resource.TestCheckFu
 	}
 }
 
-func TestAccScalewayRdbInstance_Capitalize(t *testing.T) {
```

Comment:
Why is this test deleted?

Reply:
This test was integrated in the basic test